src/Package/Impl/DataInspect/VariableGridWindowPane.cs

```diff
@@ -25,5 +27,31 @@ internal sealed class VariableGridWindowPane : RToolWindowPane {
 
             _gridHost.SetEvaluation(evaluation);
         }
+
+        #region IVsWindowFrameNotify3
+
+        public int OnClose(ref uint pgrfSaveOptions) {
+            _gridHost?.OnClose();
+
+            return VSConstants.S_OK;
+        }
+
+        public int OnDockableChange(int fDockable, int x, int y, int w, int h) {
+            return VSConstants.S_OK;
+        }
+
+        public int OnMove(int x, int y, int w, int h) {
+            return VSConstants.S_OK;
+        }
+
+        public int OnShow(int fShow) {
+            return VSConstants.S_OK;
```

Comment:
There are ways to have it show later that would not go through VE. Windows | Save Window Layout while it's visible, followed by Apply Window Layout later, for example.